static/src/javascripts/projects/common/modules/commercial/track-ad.js

```diff
@@ -0,0 +1,30 @@
+define([
+    'Promise',
+    'common/utils/mediator'
+], function(Promise, mediator) {
+    return trackAd;
+
+    function trackAd(id) {
```

Comment:
We're basically broadcasting an event over mediator, which is itself just listening to a googletag event. If this method existed inside DFP, would we even need to pass the render messages to mediator at all?